tests/integration/alertengine_test.py

```diff
@@ -3,6 +3,6 @@
 
 
 def test_all_handlers_should_be_loadable():
-    for sender in AlertSender.objects.all():
+    for sender in AlertSender.objects.filter(supported=True):
```

Comment:
This might be a good candidate for a manager or queryset-addition I am thinking. Not necessarily now, for this PR, but something to remember if we do decide on the "refactor oft used queryset dot-messes".

Reply:
Probably a good idea to make a sweep the codebase for this kind of thing. This was just introduced now, and is only used in this test and more or less once in the code, so no biggie...